openmls/src/group/mls_group/test_mls_group.rs

```diff
@@ -1144,214 +1143,1155 @@ fn remove_prosposal_by_ref(
     }
 }
 
-// Test that the builder pattern accurately configures the new group.
-#[openmls_test]
-fn group_context_extensions_proposal() {
-    let alice_provider = &mut Provider::default();
-    let bob_provider = &mut Provider::default();
-    let (alice_credential_with_key, _alice_kpb, alice_signer, _alice_pk) =
-        setup_client("Alice", ciphersuite, alice_provider);
-    let (bob_credential_with_key, _bob_kpb, bob_signer, _bob_pk) =
-        setup_client("bob", ciphersuite, bob_provider);
+mod group_context_extensions {
```

Comment:
Let's not have this in this file. Please move the tests to a new file for group context extension validation checks.